extensions/amp-story/amp-story.md

```diff
@@ -345,7 +347,7 @@ In most implementations for live blogs, content is either pushed by the server t
 
 This means that publishers of stories do not need to set up a JSON endpoint or push mechanism for this functionality to work.
 
-Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](#data-poll-interval) attribute.
+Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](<#data-poll-interval-(optional)>) attribute.
```

Comment:
```suggestion
Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](#data-poll-interval-optional) attribute.
```

Reply:
ah! thank you for the explanation on the angle brackets

on amp.dev, the parens are not being removed in the html, and as a result the anchors aren't working without them (e.g. https://amp.dev/documentation/components/amp-story/#poster-square-src-optional vs https://amp.dev/documentation/components/amp-story/#poster-square-src-(optional))